auth/src/main/kotlin/com/hedvig/auth/models/AuditEvent.kt

```diff
@@ -21,6 +21,7 @@ internal class AuditEvent(
 
     enum class EventType {
         CREATED_ON_IMPORT,
-        CREATED_ON_LOGIN
+        CREATED_ON_LOGIN,
+        CREATED_EXPLICITLY,
```

Comment:
Like the clarity! Maybe even deliberately? 😄 Regardless good naming
```suggestion
        CREATED_EXPLICITLY,
```